src/form/state/FormFieldDefinition.test.ts

```diff
@@ -0,0 +1,119 @@
+import { FormFieldDefinition } from './FormFieldDefinition.model';
+
+describe('form field description', () => {
+  describe('display and interaction with value', () => {
+    it.todo('can define the value may be treated as case insensitive');
+
+    it.todo('can define the value must be treated as case sensitive');
+
+    it('can define the value can contain multiple lines of text', () => {
+      const state: FormFieldDefinition = {
+        multiline: true,
+      };
+
+      expect(state.multiline).toBe(true);
+    });
+
+    it('can define the value should not receive spell checking suggestions', () => {
+      const state: FormFieldDefinition = {
+        spellCheck: false,
+      };
+
+      expect(state.spellCheck).toBe(false);
+    });
+
+    it('can define the can contain significant whitespace that must be preserved', () => {
```

Comment:
`it('can define the can contain significant whitespace that must be preserved'` => `it('can define the value may contain significant whitespace that must be preserved'`